db/database.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+
+SQLALCHEMY_DATABASE_URL = "sqlite:///./library.sqlite3"
+
+
+engine = create_engine(
+    SQLALCHEMY_DATABASE_URL, connect_args={"check_same_thread": False}

```

Comment:
The `connect_args` parameter with `check_same_thread: False` is specific to SQLite and allows the use of the same connection across multiple threads. Ensure this is necessary for your application's threading model.